gen3utils/manifest/validation_config.yaml

```diff
@@ -74,3 +74,15 @@ versions:
     spark: "*"
     aws-es-proxy: "*"
   desc: "tube requires spark and aws-es-proxy"
+
+
+#################################
+# GLOBAL REQUIREMENT VALIDATION #
+#################################
+# Validation for services that need global policies to work with
+
+global:
+  services:
+    - hatchery:
+        netpolicy: "on"
+      desc: "hatchery service needs netpolicy==on in the global block in the manifest"
```

Comment:
Your changes look good but with a step back, i think this check ("hatchery service needs netpolicy==on") is pretty specific. I am not sure we need to create a new generic config section like this, because i don't think we will use it again. Or if we need to validate something in the "global" block again, it might need a completely different format (because the "global" block contains very different things). We might have been better off just adding a check specifically for this hatchery check (nothing in the config, just a check in the validator code) because it'd be easier to read. But since you have already implemented it, i'll let you make that call

Reply:
I agree. I felt the same while writing it. YAGNI principle was yelling in my ears 🤣 . I wasn't really sure making a modification to the validator code would be a good approach. I was trying to be as generic as I can in the favor of code reusability. If you're certain that this scenario is not as generic as I made it to be,  then maybe it is better not to include the added complexity. I'll update the validator code accordingly.